packages/credential-showcase-traction-adapter/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../tsconfig-base.json",
```

Comment:
not now, but we can create a package with the ts-config with in the monorepo and install/share between the packages

Reply:
Not enough context for me to understand what I have to do here.